tests/orgsGetByTag.test.js

```diff
@@ -64,3 +144,52 @@ test('Get all the active organisations, associated with a specific tag', functio
     });
   });
 });
+
+test('When no orgs are found to match a specific tag', function (t) {
+  var tagId = 2;
+  init(config, function (err, server, pool) {
+    if (err) {
+      console.log('error initialise server', err);
+      return t.fail();
+    }
+
+    server.inject({
+      method: 'GET',
+      url: '/orgsGetByTag?tags=' + tagId
+    }, function (res) {
+      var expected = {
+        filter_tag: 'Corporate',
+        orgs: [{
+          id: null,
+          name: null,
+          logo_url: null,
+          active: null
+        }]
```

Comment:
I think empty array is more consistent with what we send back if there are no results. What are your thoughts?